drivers/hmis/lib/tasks/ce_starter_pack_20250302.rake

```diff
@@ -310,6 +310,17 @@ task ce_starter_pack_20250302: [:environment] do
   ce_project_funder.grant_id = 'grant ID'
   ce_project_funder.save! if ce_project_funder.changed?
 
+  # create a UnitGroup in ce project
+  unit_group = Hmis::UnitGroup.find_or_create_by(
+    project: ce_project,
+    name: 'Admin Approve Denial Unit Group',
+    workflow_template: admin_approval_template,
+  )
+  # if there are any units in the project that don't have a unit group, assign them to this one
+  ce_project.units.where(unit_group: nil).each { |u| u.update!(unit_group: unit_group) }
+  # clean up any dangling opportunities that don't have an owner
+  Hmis::Ce::Opportunity.all.filter { |o| o.owner.nil? }.each { |u| u.referrals.each(&:destroy!) }.map(&:destroy!)
```

Comment:
Hm so the first time I ran locally this I got an error `ActiveRecord::DeleteRestrictionError: Cannot delete record because of dependent referrals` But the second time it succeeded. Weird, maybe it just needs a judicious `reload` or something so it "knows" it doesn't have any referrals left after they have all been destroyed? 

Anyway, I think this is no big deal since it's just a starter pack and not production code. But we may have to double-run it in QA too.